pixi/src/checkAggregation/statusBanner.js

```diff
@@ -65,11 +65,13 @@ const getStatusId = async (
       safeBrowsing.safeBrowsing &&
       linter.usesHttps;
 
-    const pagePassedAll = pageExperience.isAllFast && passedOtherCriteria;
-
-    if (cacheExperience) {
-      const cachePassedAll = cacheExperience.isAllFast && passedOtherCriteria;
+    const dataType = pageExperience.summary.source;
+    const pagePassedAll =
+      pageExperience.summary.isAllFast[dataType] && passedOtherCriteria;
```

Comment:
any reason to do a null check on `pageExperience.summary`?

Reply:
No, but ended up refactoring to remove `summary` anyway.